tests/gtest/avifclaptest.cc

```diff
@@ -129,27 +131,46 @@ constexpr ValidClapPropertyParam kValidClapPropertyTestParams[] = {
      100,
      AVIF_PIXEL_FORMAT_YUV420,
      {99, 1, 99, 1, static_cast<uint32_t>(-1), 2, static_cast<uint32_t>(-1), 2},
-     {0, 0, 99, 99}},
+     {0, 0, 99, 99},
+     false},
+    // pcX = -1/2 + (100 - 1)/2 = 49
+    // pcY = -1/2 + (100 - 1)/2 = 49
+    // leftmost = 49 - (99 - 1)/2 = 0
+    // topmost = 49 - (99 - 1)/2 = 0
```

Comment:
This comment is exactly the same as the comment for the previous test case (lines 126-129), so this comment must be incorrect.

I think this comment should read:

```
    // pcX = 1/2 + (100 - 1)/2 = 50
    // pcY = 1/2 + (100 - 1)/2 = 50
    // leftmost = 50 - (99 - 1)/2 = 1
    // topmost = 50 - (99 - 1)/2 = 1
```